### ISSUE TITLE
Causal Bayesian Networks

### ISSUE THREAD

**Issue by ntanmayee** [pgmpy/pgmpy#676]
Are there any plans for implementing causal Bayesian networks?

**Comment by ankurankan** [comment]
We don't have any plan for implementing causal networks in our current
roadmap. But I think it should be fairly easy to work with causal networks
using BayesianModel class. Let me know if you have any doubts.

**Comment by ntanmayee** [comment]
Thanks for your quick response! 

Yes, BayesianModel class does seem to give some basic functionality of causal Bayesian Networks. However, I was referring to algorithms specifically for causal discovery and for estimating intervention effects. 

Would it be okay if I worked on these features independently? Do let me know!

**Comment by ankurankan** [comment]
Yeah, that would be great to have in pgmpy. Please let me know if you need
any help.

Ankur

**Comment by Manas-7854** [comment]
Hi @ankurankan, I was thinking of taking up this issue, can you elaborate on what needs to be done

**Comment by ankurankan** [comment]
This is already done, sorry we forgot to close it. `pgmpy.inference.CausalInference` implements functionality for Causal Bayesian Networks.
